staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+ * Web Components lib
+ * 
+ * @author manuelbarzi
```

Comment:
Ohhh my god!!! This code was made by Manuel Barzi, not you!!!

Reply:
Yes i know it ! That is his version. My version is the 0.0.1. However, should i change the author name?